packages/vite/src/node/plugins/manifest.ts

```diff
@@ -19,6 +19,10 @@ interface ManifestChunk {
   dynamicImports?: string[]
 }
 
+function isEmptyChunk(chunk: OutputChunk): boolean {
+  return !chunk.code || (chunk.code.length <= 1 && !chunk.code.trim())
```

Comment:
In the cases I observed, `code` was always `\n`, so there's potential for simplification here.

Didn't want to run `trim()` on a large string, hence the check for `length`.